Release 22.06.0 (#6246)
scalableminds/webknossos: CHANGELOG.released.md

and this project adheres to [Calendar Versioning](http://calver.org/) `0Y.0M.MICRO`.
For upgrade instructions, please check the [migration guide](MIGRATIONS.released.md).

## [22.06.0](https://github.com/scalableminds/webknossos/releases/tag/22.06.0) - 2022-05-27
[Commits](https://github.com/scalableminds/webknossos/compare/22.05.1...22.06.0)

### Highlights

> - Added a Volume Interpolation feature. When triggering the interpolation, the current segment id is interpolated between the current slice and the slice which was annotated last. Note that the feature is forbidden for tasks by default, but can be enabled/recommended. The feature can be used via a button in the toolbar or via the shortcut V. [#6162](https://github.com/scalableminds/webknossos/pull/6162) and [#6235](https://github.com/scalableminds/webknossos/pull/6235)

Reviewer: ```suggestion
- Added a volume interpolation feature. When triggering the interpolation, the current segment id is interpolated between the current slice and the slice which was annotated last. Note that the feature is forbidden for tasks by default, but can be enabled/recommended. The feature can be used via a button in the toolbar or via the shortcut V. [#6162](https://github.com/scalableminds/webknossos/pull/6162) and [#6235](https://github.com/scalableminds/webknossos/pull/6235)
```

- Removed the feature to copy a segment from the previous/next slice with the V shortcut. Use the new volume interpolation feature instead (also bound to V and available via the toolbar). [#6235](https://github.com/scalableminds/webknossos/pull/6235)

> - Selecting "Download" from the tracing actions now opens a new modal, which lets the user select data for download, start TIFF export jobs or copy code snippets to get started quickly with the webKonossos Python Client. [#6171](https://github.com/scalableminds/webknossos/pull/6171)

Reviewer: ```suggestion
- Selecting "Download" from the annotation actions now opens a new modal, which lets the user select data for download, start TIFF export jobs or copy code snippets to get started quickly with the webKonossos Python Client. [#6171](https://github.com/scalableminds/webknossos/pull/6171)
```

> - Added a long-running job that applies the merging done via a merger mode tracing to a new output dataset. The job is accessible via a button next to the merger mode button once the merger mode is active. [#6086](https://github.com/scalableminds/webknossos/pull/6086)

Reviewer: ```suggestion
- Added a long-running job that materializes annotations as datasets. It merges volume annotations with their base segmentation layer and it applies “merger mode” annotations, allowing to agglomerate segments. Access this feature from the layers sidebar. When merger mode is active, an additional button is shown in the tool bar. [#6086](https://github.com/scalableminds/webknossos/pull/6086)
```
